### DISCUSSION TITLE
Is roadmap/vote features site up to date?

### DISCUSSION THREAD

**Discussion by TOKN2** [microsoft/WindowsAppSDK#3518] [Ideas] [answered]
From WindowsAppSDK github I found this site: https://portal.productboard.com/winappsdk/1-windows-app-sdk/tabs/7-backlog . 
The site doesn't seem to be working.

Is it up to date?
What is an alternative where we can see planned progress?

**Comment by wjk** [comment, marked as answer]
As I understand it, the site you linked was set up using the personal account of an individual who is no longer employed by Microsoft. Microsoft no longer has any access to that site and it has been abandoned.

**Reply by teohumeau** [reply]
The roadmap link at https://github.com/microsoft/WindowsAppSDK#helpful-links should be updated or removed if this site isn't used anymore.
